tests-beta/android/MASVS-CODE/MASTG-TEST-0290.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Enforced Application Updates
```

Comment:
It would be great to understand from the title what to check as a tester about the Enforced App Updates. Is it testing, enforcing, validating, ...? 

https://docs.google.com/document/d/1EMsVdfrDBAu0gmjWAUEs60q-fWaOmDB5oecY9d9pOlg/edit?tab=t.0#heading=h.42oc9ti8yxnx